index.js

```diff
@@ -5,18 +5,18 @@ document.addEventListener('DOMContentLoaded', () => {
 
     // Afegeix un listener d'esdeveniments per a la presentació del formulari
     form.addEventListener('submit', (event) => {
-        event.preventDefault(); // Prevenir el comportament predeterminat de presentació del formulari
+        event.preventDefault(); 
 
         // Recupera les dades del formulari
-        const title = sanitizeInput(form.noteTitle.value);
-        const content = sanitizeInput(form.noteContent.value);
+        const title = form.noteTitle.value;
+        const content = form.noteContent.value;
         const priority = form.noteSelect.value;
 
         // Valida la selecció de prioritat
         if (priority === "Select priority") {
             console.log("No priority");
             alert("No s'ha seleccionat cap prioritat. Si us plau, seleccioneu una prioritat.");
-            return; // Sortir de la funció anticipadament si no s'ha seleccionat cap prioritat
```

Comment:
This return avoids to render `No priority` when it is not selected. Did u check it?

- [ ] Fix this missing feature.